recipes/libpng/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.6.21" %}
```

Comment:
Not that I can pinpoint something that says this is ok yet, but the feeling from what I have read is that it is ok. So, I'm not worried about this.